client/packages/invoices/src/InboundShipment/api/api.ts

```diff
@@ -233,6 +241,37 @@ export const getInboundQueries = (sdk: Sdk, storeId: string) => ({
       },
     });
   },
+  updateServiceTax: async ({
+    lines,
+    tax,
```

Comment:
```suggestion
    taxPercentage,
```
Just to simplify the update below and to make the intention clearer